libs/sdk-ui-kit/src/DatePicker/styles/Datepicker.scss

```diff
@@ -0,0 +1,257 @@
+// (C) 2007-2020 GoodData Corporation
+@import "../../../styles/scss/mixins.scss";
+
+/* ==========================================================================
+   Datepicker
+   ========================================================================== */
+.gd-datepicker {
+    position: relative;
+
+    .input-text {
+        position: relative;
+        padding-left: 36px;
+
+        @include transition("padding-left", 0);
+    }
+
+    &.small {
+        .input-text {
+            padding-left: 30px;
+        }
+
+        .gd-datepicker-icon {
+            left: 8px;
+            line-height: 28px;
+            font-size: 15px;
+        }
+    }
+
+    &.gd-datepicker-focused {
+        .input-text,
+        .gd-datepicker-icon {
+            z-index: 11;
+        }
+    }
+}
+
+.gd-datepicker-icon {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 10px;
+    line-height: 30px;
+    color: $gd-color-highlight;
```

Comment:
I am missing the definition of css variable somewhere

Reply:
Added `$gd-date-picker-color: var(--gd-palette-primary-base, $gd-color-highlight);`